src/classes/CON_Preferred_TEST.cls

```diff
@@ -37,92 +37,116 @@
-			WorkPhone__c = '555-432-4433', PreferredPhone__c = 'Work');
-		Contact contact3 = new Contact(LastName = 'TestersonC', AlternateEmail__c = 'fake@test3.com', Preferred_Email__c = 'Alternate', 
-			OtherPhone = '555-432-4433', PreferredPhone__c = 'Other');
-		Contact contact4 = new Contact(LastName = 'TestersonD', AlternateEmail__c = 'fake@test4.com',Preferred_Email__c = 'Alternate',
-			MobilePhone = '555-432-4433', PreferredPhone__c = 'Mobile');
-			
-		List<Contact> contacts = new Contact[]{contact1, contact2, contact3, contact4};
-		Test.startTest();
-		insert contacts;
-		Test.stopTest();
-		
-		contacts = [SELECT Email, LastName, Phone, UniversityEmail__c, WorkEmail__c, AlternateEmail__c, HomePhone, WorkPhone__c, OtherPhone, MobilePhone 
-					FROM Contact where ID in :contacts];
-		
-		//ensure we can retrieve the contacts in a specific order/know which contact we are comparing
-		//can no longer user Name in filters if we want to be Shield compatible
-		Map<String, Contact> contactsByLastName = new Map<String, Contact>();
-		for(Contact c : contacts) {
-			contactsByLastName.put(c.LastName, c);
-		}
-		contact1 = contactsByLastName.get('TestersonA');
-		contact2 = contactsByLastName.get('TestersonB');
-		contact3 = contactsByLastName.get('TestersonC');
-		contact4 = contactsByLastName.get('TestersonD');
-		
-		System.assertEquals(contact1.UniversityEmail__c, contact1.Email);
-		System.assertEquals(contact2.WorkEmail__c, contact2.Email);
-		System.assertEquals(contact3.AlternateEmail__c, contact3.Email);
-		
-		System.assertEquals(contact1.HomePhone, contact1.Phone);
-		System.assertEquals(contact2.WorkPhone__c, contact2.Phone);
-		System.assertEquals(contact3.OtherPhone, contact3.Phone);
-		System.assertEquals(contact4.MobilePhone, contact4.Phone);
-	}
-	
-	@isTest
-	public static void preferredEmailPhoneUpdate() {
-		
-		// Preferred email is required to pass checks in preferred trigger
-		Contact contact1 = new Contact(LastName = 'TestersonE', UniversityEmail__c = 'fake@test5.com', HomePhone = '555-432-4433',Preferred_Email__c = 'University');
-		Contact contact2 = new Contact(LastName = 'TestersonF', WorkEmail__c = 'fake@test6.com', WorkPhone__c = '555-432-4433',Preferred_Email__c = 'Work');
-		Contact contact3 = new Contact(LastName = 'TestersonG', AlternateEmail__c = 'fake@test7.com', OtherPhone = '555-432-4433',Preferred_Email__c = 'Alternate');
-		Contact contact4 = new Contact(LastName = 'TestersonH', MobilePhone = '555-432-4433');
-			
-		List<Contact> contacts = new Contact[]{contact1, contact2, contact3, contact4};
-		insert contacts;
+    @isTest
+    public static void preferredEmailPhoneInsert() {
+    
+        Test.testInstall(new STG_InstallScript(), null);
+        List<trigger_handler__c> preferredPhoneHandler = [select id, class__c, active__c from trigger_handler__c WHERE class__c = 'CON_PreferredPhone_TDTM' AND Object__c = 'Contact'];
```

Comment:
Can we make the syntax here all caps for the SOQL commands, "SELECT...FROM...WHERE...AND"

Reply:
Minor, but I don't see this done.

Reply:
@skosgi1 can we clean this part up? (see above comments, again minor but low effort)